website/www/site/content/en/documentation/patterns/overview.md

```diff
@@ -56,7 +56,7 @@ Pipeline patterns demonstrate common Beam use cases. Pipeline patterns are based
 
 ## Contributing a pattern
 
-To contribute a new pipeline pattern, create an issue with the [`pipeline-patterns` label](https://issues.apache.org/jira/browse/BEAM-7449?jql=labels%20%3D%20pipeline-patterns) and add details to the issue description. See [Get started contributing](/contribute/) for more information.
+To contribute a new pipeline pattern, create [a feature request](https://github.com/apache/beam/issues/new?assignees=octocat&labels=new+feature%2Cawaiting+triage&template=feature.yml&title=%5BFeature+Request%5D%3A+) and add details to the issue description. See [Get started contributing](/contribute/) for more information.
```

Comment:
this looks broken

Reply:
This one was actually correct, though the octocat piece wasn't needed (it doesn't harm anything though)